tests/http_proxies/run.sh

```diff
@@ -49,13 +50,18 @@ function prepare() {
     start_tidb_cluster --workdir $WORK_DIR
 
     start_proxy
+    # TODO: make sure the proxy is started.
+    sleep 20
```

Comment:
Could you add some retry mechanism for proxy checking instead of sleep 20s

Reply:
done